src/styles/blocks/header.scss

```diff
@@ -0,0 +1,20 @@
+.header {
+  box-shadow: 0 2px 4px 0 #0000000d;
+
+  @include font;
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  padding: 0 50px;
```

Comment:
Consider using relative units like `rem` or `em` for padding instead of fixed units like `px` to improve responsiveness and scalability.